tests/cms/views/push_notifications/test_push_notification_form_view.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from django.test.client import Client
+from django.urls import reverse
+
+from integreat_cms.cms.models import Language, Region
+
+
+@pytest.mark.django_db
+def test_validate_forms(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+) -> None:
+    client, role = login_role_user
+
+    #    default_language = Language.objects.create(name="Deutsch")
+    #    region = Region.objects.create(name="Augsburg", default_language=default_language)
+    edit_push_notification = reverse(
+        "edit_push_notification",
+        kwargs={
+            "push_notification_id": 1,
+            "region_slug": "Augsburg",
```

Comment:
```suggestion
            "region_slug": "augsburg",
```
The slug has to be lower capital. If you adapt it this way, some of the test fixtures will already pass :)